docs/_adapters/istio/istio.md

```diff
@@ -6,13 +6,26 @@ mesh_name: Istio
 earliest_version: v1.6.0
 port: 10000/gRPC
 project_status: stable
-adapter_version: v0.5.4
 lab: istio-meshery-adapter
 github_link: https://github.com/meshery/meshery-istio
 image: /assets/img/service-meshes/istio.svg
 permalink: service-meshes/adapters/istio
 ---
 
+{% assign sorted_tests_group = site.compatibility | group_by: "meshery-component" %}
+{% for group in sorted_tests_group %}
+      {% if group.name == "meshery-istio" %}
+        {% assign items = group.items | sort: "meshery-component-version" | reverse %}
+        {% for item in items limit: 1 %}
+          {% if item.meshery-component-version != "edge" %}
+            {% if item.overall-status == "passing" %}
```

Comment:
Please add a condition for status failing too, in case of failing the adapter version should return the previous latest passing status

Reply:
hey @asubedy ,i'm trying this code but its not working ,could you suggest why .
```
{% assign sorted_tests_group = site.compatibility | group_by: "meshery-component" %}
{% for group in sorted_tests_group %}
      {% if group.name == "meshery-istio" %}
        {% assign items = group.items | sort: "meshery-component-version" | reverse %}
        {% for item in items %}
          {% if item.meshery-component-version != "edge" %}
            {% if item.overall-status == "passing" %}
              {% assign adapter_version_dynamic = item.meshery-component-version %}
              {% break %}
            {% elsif item.overall-status == "failing" %}
              {% continue %}
            {% endif %}
          {% endif %}
        {% endfor %} 
      {% endif %}
{% endfor %}
```

Reply:
You can discard the for loop, since ` items` will always have reverse sorted data that means items[0] will always have latest one so if test is passing set `adapter_version_dynamic` to `items[0]` else to `items[1]`

Reply:
hey @asubedy ,now the above code snippet is working, there was a small bug which i fixed.
Also i made a new commit having that code .If its fine with you ,i can make the same changes for other adapters.